src/api/democracy.ts

```diff
@@ -19,6 +20,26 @@ const CONVICTIONS_BLOCKS: { [key: string]: number } = {
   locked6x: 1612800,
 }
 
+const voteConviction = (vote?: PalletDemocracyVoteAccountVote): String => {
+  if (vote?.isSplit) {
+    return 'None'
+  } else if (vote?.isStandard) {
+    return vote.asStandard?.vote.conviction.toString()
+  } else {
+    return 'None'
+  }
```

Comment:
```suggestion
if (vote?.isStandard) {
    return vote.asStandard?.vote.conviction.toString()
  } else {
    return undefined
  }
```

Reply:
what about when the vote is split? the conviction should be defined at: `None`:

```typescript
    interface PalletDemocracyConviction extends Enum {
        readonly isNone: boolean;
        readonly isLocked1x: boolean;
        readonly isLocked2x: boolean;
        readonly isLocked3x: boolean;
        readonly isLocked4x: boolean;
        readonly isLocked5x: boolean;
        readonly isLocked6x: boolean;
        readonly type: 'None' | 'Locked1x' | 'Locked2x' | 'Locked3x' | 'Locked4x' | 'Locked5x' | 'Locked6x';
    }
 ```

In the conviction in the runtime the conviction for split vote is default:
https://github.com/galacticcouncil/hydration-node/blob/769c33d63d24356791c2f0e276350ebdc2914005/pallets/staking/src/integrations/democracy.rs#L53

Reply:
it doesn't matter since None doesn't have any value and you won't get any action points

Reply:
but yeah, let's leave None

Reply:
@thiolliere will you apply that suggestion? I changed it to 'None'